packages/affiliates/test/mocha/devmining-v2.js

```diff
@@ -0,0 +1,321 @@
+const { DevMining } = require("../../libs/affiliates");
```

Comment:
this copies the tests from v1, changes some values, but does a good job of testing scenarios